examples/README.md

```diff
@@ -56,6 +57,13 @@ Therefore, it is essential for the example contributors to know how to integrate
 3. Export your dataset path with the prefix `/data` and make sure you have a copy of the dataset in the `/data/scratch/examples-data` directory on the CI machine. Community contributors can contact us via slack to request for downloading the dataset on the CI machine.
 4. Implement the logic such as dependency setup and example execution
 
+## Benchmark Your Example
```

Comment:
This content is outdated and should be moved to `language/bert/README.md`

Reply:
Please add a readme under `language/bert` including `setup`, `usage` and `result` for both finetuning example and benchmark.